laura/portfolio/src/main/webapp/index.html

```diff
@@ -6,27 +6,34 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  
+    <ul>
+        <li><a href="index.html" class="navbar">Home</a>
+            <a href="mood.html" class="navbar">Mood generator</a></li>
+    </ul>
+
   <body>
     <h1 style="text-align:center;">My Portfolio</h1>
+    <h2> Bio </h2>
 
     <div class="row">
         <div class="column">
-            <p> Hello, there! I am Laura, I'm 19yo and I live in Bucharest, Romania.
-            Since 2019, I study Computer Science at University College, Oxford (the oldest college in the university, founded in 13th century).</p>
+            <p> Hello, there! I am Laura, I'm 19yo and I am from Bucharest, Romania.</p>
+            <p>Since 2019, I study Computer Science at University College, Oxford (the oldest college in the university, founded in 13th century).</p>
             
-            <img src="images/univ.jpg" height="300"/>  
+            <img src="images/univ.jpg" height="400" align="center"/>  
```

Comment:
maybe use % so it looks good on all screen size?

Reply:
solved in new commit.